precious-core/src/config.rs

```diff
@@ -391,6 +402,8 @@ impl Config {
     }
 }
 
+const DEFAULT_LABEL: &str = "default";
```

Comment:
(nit) I would stick this above the `impl Config` line; I know rustc doesn't care but reading into_commands, I found myself wondering where it came from.